django_project/changes/views/category.py

```diff
@@ -127,7 +128,7 @@ def get_queryset(self):
         return qs
 
 
-class CategoryListView(CategoryMixin, StaffuserRequiredMixin, ListView):
+class CategoryListView(CategoryMixin, LoginRequiredMixin, ListView):
```

Comment:
Might good to follow this pattern: https://docs.djangoproject.com/en/2.0/topics/auth/default/#the-loginrequired-mixin

> This mixin should be at the leftmost position in the inheritance list.

https://stackoverflow.com/a/35746181/2395485

Maybe you can check them all?